packages/graphql-language-service-server/src/MessageProcessor.ts

```diff
@@ -68,6 +69,16 @@ import { GraphQLExtensionDeclaration } from 'graphql-config';
 import type { LoadConfigOptions } from './types';
 import { promisify } from 'util';
 
+// It would be far better to use nodes native url.fileURLToPath but
+// its not supported before node v10.12.0.
```

Comment:
we support node v10.17+ because dynamic import

Reply:
Only to be sure. That means the minimal node version is node v10.17. Would it be ok if I would add the minimal version to package.json?

Reply:
we had `engines` defined for some packages before, but it caused too many downstream issues

Reply:
I switched the impl to use the native fileURLToPath.